win32port/zlib/gzguts.h

```diff
@@ -12,7 +12,7 @@
 #  endif
 #endif
 
-#if ((__GNUC__-0) * 10 + __GNUC_MINOR__-0 >= 33) && !defined(NO_VIZ)
+#if ((__GNUC__-0) * 10 + __GNUC_MINOR__-0 >= 33) && !defined(NO_VIZ) && !defined(_WIN32)

```

Comment:
Without it, I run into the following warning:

In file included from w:/libwebsockets/win32port/zlib/gzio.c:13:0:
w:/libwebsockets/win32port/zlib/gzguts.h:16:0: warning: "ZLIB_INTERNAL" redefined [enabled by default]
 #  define ZLIB_INTERNAL __attribute__((visibility ("hidden")))
 ^
In file included from w:/libwebsockets/win32port/zlib/gzio.c:12:0:
w:/libwebsockets/win32port/zlib/zutil.h:19:0: note: this is the location of the previous definition
 #  define ZLIB_INTERNAL
 ^

Could be, that there is a better solution to fixing this issue.